README.md

```diff
@@ -1,284 +1,305 @@
 
+If this is not set, the default maximum number of accounts is 4. The request will be rejected if an attempt is made that causes it to go over this limit.
 
-  * jacoco
+## List Sensitivity checking
 
-    https://docs.gradle.org/current/userguide/jacoco_plugin.html
+### Verified Users
 
-    Provides code coverage metrics for Java code via integration with JaCoCo.
-    You can create the report by running the following command:
+If a publication is PRIVATE or CLASSIFIED, account management is used to validate whether the user has permission to see the publication.
 
-    ```bash
-      ./gradlew jacocoTestReport
-    ```
+For PRIVATE, as long as the user exists in our database, then they are able to see the publication.
 
-    The report will be created in build/reports subdirectory in your project directory.
+For CLASSIFIED, rules are applied based on the List Type and Provenance of the user. For reference, the provenance is which IDAM the user came from (PI_AAD, CFT_IDAM or CRIME_IDAM). The mappings of which provenance can see which list type is defined in the [enum file](./src/main/java/uk/gov/hmcts/reform/pip/account/management/model/ListType.java)
 
-  * io.spring.dependency-management.
+### Third Party Users
 
-    https://github.com/spring-gradle-plugins/dependency-management-plugin
+For third party users, a similar process is applied however is based on List Type + Third Party role. This is also defined in: [enum file](./src/main/java/uk/gov/hmcts/reform/pip/account/management/model/ListType.java)
 
-    Provides Maven-like dependency management. Allows you to declare dependency management
-    using `dependency 'groupId:artifactId:version'`
-    or `dependency group:'group', name:'name', version:version'`.
+## Account Application Process
 
-  * org.springframework.boot
+When a user applies for a new PI_AAD verified account they go through an approval process. Account management deals with all CRUD operations associated with this process.
 
-    http://projects.spring.io/spring-boot/
+On Approval / Rejection, the uploaded ID file will be deleted.
 
-    Reduces the amount of work needed to create a Spring application
+## Expired / Inactive accounts
 
-  * org.owasp.dependencycheck
+For all users except for third party, there is an inactive account process to ensure accounts are deleted when they are no longer used.
 
-    https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/index.html
+For IDAM and Admin users, this uses the sign in date. For internal, this will use the verified date.
 
-    Provides monitoring of the project's dependent libraries and creating a report
-    of known vulnerable components that are included in the build. To run it
-    execute `gradle dependencyCheck` command.
+Users will be sent a reminder to login / re-verify 14 days before their account is due to expire.
 
-  * com.github.ben-manes.versions
+If they do not complete this process by the 14 days, their account will be deleted.
```

Comment:
May be then sentence **should** be ".....complete this process **within** 14 days...."?